Cargo.toml

```diff
@@ -11,13 +11,13 @@ keywords = ["cargo", "patch", "dependency", "dependencies", "patchfile"]
 categories = ["command-line-utilities"]
 
 [dependencies]
-anyhow = "1"
-cargo = "0.72"
-fs_extra = "1"
+anyhow = "1.0.89"
```

Comment:
I would prefer if you use the same version formatting as I did before.

1.x.y -> 1
0.1.x -> 0.1